script.js

```diff
@@ -11,15 +11,28 @@ var publikationsDatum = getCurrentDate(); //setting a current date
 
 function fillTableFromDict(summaryTableData) {
   for (let elementId in summaryTableData) {
-    const value = summaryTableData[elementId];
-    document.getElementById(elementId).innerText = value;
+    document.getElementById(elementId).innerText = summaryTableData[elementId];
   }
 }
 
+function getEUSummary(allCountries) {
+  var EUCodes = Object.keys(EUCountries).map(key => EUCountries[key]);
+  EUCodes = EUCodes.concat(Object.keys(EEACountries).map(key => EEACountries[key]));
```

Comment:
Strange...if everything is EU => you can just list all counties in one array and get codes in one go, without needs to concat the objects.

Reply:
It was because I have 
![image](https://user-images.githubusercontent.com/7361924/97607900-e72a3000-1a11-11eb-8823-687154c2c0ef.png)
EU / EEA / UK - countries hier, in EEA there are 3 more countries, that are not in EU, and + UK then. I didn't want to add then all to the one dictionary, that's the reason, why I have to concate the lists afretwards.